routes/index.js

```diff
@@ -219,33 +224,36 @@ function getTasksWithTaskDefinitions(cluster) {
               // Batch the batches :) - describe up to 2 batches of batches of maxSize ARNs at a time
               // Without batchPromises, we will fire all ecs.describeTasks calls one after the other and could run into API rate limit issues
               return batchPromises(2, taskBatches, taskBatch => new Promise((resolve, reject) => {
-                // The iteratee will fire after each batch
-                //console.log("\nCalling ecs.describeTasks for", taskBatch);
-                resolve(ecs.describeTasks({cluster: cluster, tasks: taskBatch}).promise());
+                  // The iteratee will fire after each batch
+                  debugLog(`\tCalling ecs.describeTasks for Task batch: ${taskBatch}`);
+                  resolve(ecs.describeTasks({cluster: cluster, tasks: taskBatch}).promise()
+                      .then(delayPromise(1000)));
```

Comment:
Would it be better to catch the rate exceeded error and delay at that point?

Reply:
Thanks @jlambert121. 
One reason to pre-emptively throttle our ECS API usage here is that we want to be a good citizen and prevent other clients of the ECS SDK operating simultaneously from suffering from rate limit issues.
I'll add a TODO to make the delay timing configurable.

Reply:
@jlambert121 I've moved this PR to https://github.com/ExpediaDotCom/c3vis/pull/26 to clean up the commit history. Feel free to add further comments there.